Index.html

```diff
@@ -1,53 +1,62 @@
-       <h1>about</h1>
-       <h5>WE CREATE DELICIOUS MEMORIES</h5>
-       <p>Renome it is renowned restaurant delights the senses with elegant ambiance, gracious service and delectable menus in the heart of Manhattan’s. The distinctive setting boasts contemporary furnishings and finishes atop the restaurant’s grand modern architecture. Renome offers the room for private dining where guests enjoy creative foods and sumptuous late night desserts. Is one of the few Manhattan restaurants to have a coveted 3-star Michelin rating.</p>
-   </div>
-   <div class="container-gallery">
-       <h1 class="gallery">gallery</h1>
-       <h5 class="gallery">MORE ABOUT US VIA PHOTOS</h5>
-       <div class="flex-container">
-           <img src="/images/gallery_img_1.png" alt="gallery1">
-           <img src="/images/gallery_img_3.png" alt="gallery2">
-           <img src="/images/gallery_img_2.png" alt="gallery3">
-           <img src="/images/gallery_img_6.png" alt="gallery4">
-           <img src="/images/gallery_img_5.png" alt="gallery5">
-           <img src="/images/gallery_img_4.png" alt="gallery6">
-       </div>
-   </div>
-   <div class="container-footer">
-    <h5>FOLLOW US:</h5>
-    <div class="footer-logo">
-        <a href="https://www.facebook.com/"><img src="/images/657794_chat_circles_facebook_line_neon_icon.png"></a>
-        <a href="https://twitter.com"><img src="/images/657766_circles_line_neon_social_tweet_icon.png"></a>
-        <a href="https://google.com"><img src="/images/657788_circles_google_google+_line_neon_icon.png"></a>
+    <div class="container-home">
+        <h4>made with love</h4>
+        <p>for you</p>
+        <div>
+            <button class="button left"><a class="arrow">&#8249;</a></button>
+            <button class="button right"><a class="arrow">&#8250;</a></button>
+        </div>
     </div>
-    <div class="footer-logo">
-        <h5>Renome</h5>
+    <div class="about">
+        <div class="about-image">
+            <img class="about-back-image" src="/images/about_img_2.png" alt="house-about1">
+            <img class="about-top-image" src="/images/about_img_1.png" alt="house-about2">
+        </div>
+        <h1 class="about-title">about</h1>
+        <h5 class="about-subtitle">We create delicious memories</h5>
+        <p class="about-text">Renome it is renowned restaurant delights the senses with elegant ambiance, gracious
+            service and
+            delectable menus in the heart of Manhattan’s. The distinctive setting boasts contemporary furnishings
+            and finishes atop the restaurant’s grand modern architecture. Renome offers the room for private dining
+            where guests enjoy creative foods and sumptuous late night desserts. Is one of the few Manhattan
+            restaurants to have a coveted 3-star Michelin rating.</p>
+    </div>
+    <div class="gallery">
+        <h1 class="gallery-text">gallery</h1>
+        <h5 class="gallery-text-more">More about us via photos</h5>
```

Comment:
this could have class gallery-subheading

Reply:
Changed